core/controllers/android_e2e_config.py

```diff
@@ -79,11 +81,13 @@ def post(self):
 
         if not constants.DEV_MODE:
             raise Exception('Cannot load new structures data in production.')
-        if topic_services.does_topic_with_name_exist(
-                'Android test'):
-            topic = topic_fetchers.get_topic_by_name('Android test')
+        if topic_services.does_topic_with_name_exist('Android test'):
+            topic = topic_fetchers.get_topic_by_name(
+                'Android test', strict=True
```

Comment:
This is a functional change from before--is this needed?

Reply:
> is this needed?

Yes, because strict=false allows None values, whereas strict=True only allows non-None values and on None values, an explicit error is thrown. And if we see the flow of code then it seems to me that a None topic is not expected here. 

Let me know if this clears things up. I'm happy to explain a bit further if needed.